css/ko.css

```diff
@@ -4,6 +4,10 @@
 */
 /* For image callouts in writing-middleware.md */
 
+@import url('../fonts/ko/Pretendard.css');
+
+html, body {font-family: Pretendard;}
```

Comment:
It's advisable to add at least one alternative font besides Pretendard, although I believe it wouldn't be necessary in this case as there's already an alternative in the main styles. Am I correct?

Reply:
@bjohansebas Correct. I'll add Arial, sans-serif font for alternative.